geospaas_harvesting/harvest.py

```diff
@@ -60,9 +59,20 @@ def _validate(self):
             assert self.HARVESTER_CLASS_KEY in config.keys(), (
                 "Harvester configuration must contain the following keys: " +
                 ', '.join(self.HARVESTER_CLASS_KEY))
-
             assert isinstance(config['class'], str), (
                 f"In '{name}' section: 'class' must be a string")
+            if 'password' in config:
+                assert config['password'] != None, "An environmental variable (not a None) is needed\
+                that has been previousely set with the same name in front of 'password' section of \
+                    configuration of this harvester"
```

Comment:
We either need a generic way to validate harvesters parameters, of leave it as it is and rely on the exceptions raised in the rest of code.

Also, please avoid breaking lines using `\` as much as possible (see [PEP8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length)).

Reply:
That was a very long string which can't we separated into several line unless we use the `\` or we use the `"""` for it. Done in the next commit.